rbi/bible_bot.rbi

```diff
@@ -0,0 +1,213 @@
+    def integer_id(book_id:, chapter_number:, verse_number:); end
+  end
+
+  sig { returns(BibleBot::Book) }
+  def book; end
+
+  sig { returns(Integer) }
+  def chapter_number; end
+
+  sig { returns(Integer) }
+  def verse_number; end
+
+  sig { returns(Integer) }
+  def id; end
+
+  sig { returns(String) }
+  def string_id; end
+
+  sig { params(other: BibleBot::Verse).returns(Integer) }
+  def <=>(other); end
+
+  sig { params(include_book: T::Boolean, include_chapter: T::Boolean, include_verse: T::Boolean).returns(String) }
+  def formatted(include_book: true, include_chapter: true, include_verse: true); end
+
+  sig { returns(T.nilable(BibleBot::Verse)) }
+  def next_verse; end
+
+  sig { returns(T::Boolean) }
+  def last_verse_in_chapter?; end
+
+  sig { returns(T::Boolean) }
+  def last_chapter_in_book?; end
+
+  sig { returns(T::Boolean) }
+  def valid?; end
+
+  sig { void }
+  def validate!; end
+end
+
+class BibleBot::Reference
+  class << self
+    sig { params(start_verse_id: Integer, end_verse_id: T.nilable(Integer)).returns(BibleBot::Reference) }
+    def from_verse_ids(start_verse_id, end_verse_id = nil); end
+
+    sig { params(text: T.nilable(String), validate: T::Boolean).returns(BibleBot::References) }
+    def parse(text, validate: false); end
+
+    sig { params(text: T.nilable(String)).returns(String) }
+    def normalize(text); end
+
+    sig { params(text: T.nilable(String)).returns(T::Array[String]) }
+    def normalize_by_chapter(text); end
+  end
+
+  sig { returns(T::Verse) }
+  def start_verse; end
+
+  sig { returns(T::Verse) }
+  def end_verse; end
+
+  sig { returns(T::Boolean) }
+  def contains_apocrypha?; end
+
+  sig { returns(String) }
+  def formatted; end
+
+  sig { returns(T::Boolean) }
+  def same_start_and_end_book?; end
+
+  sig { returns(T::Boolean) }
+  def same_start_and_end_chapter?; end
+
+  sig { returns(T::Boolean) }
+  def full_chapters?; end
+
+  sig { params(verse: BibleBot::Verse).returns(T::Boolean) }
+  def includes_verse?(verse); end
+
+  sig { params(other: BibleBot::Reference).returns(T::Boolean) }
+  def intersects_reference?(other); end
+
+  sig { returns(T::Array[BibleBot::Verse]) }
+  def verses; end
+
+  sig { returns(T::Boolean) }
+  def valid?; end
+
+  sig { void }
+  def validate!; end
+end
+
+class BibleBot::References
+  include Enumerable
+  extend T::Generic
+  Elem = type_member(:out) { { fixed: BibleBot::Reference } }
+
+  sig { returns(T::Array[Integer]) }
+  def ids; end
+
+  sig { returns(T.nilable(String)) }
+  def formatted; end
+
+  sig { returns(T::Boolean) }
+  def contains_apocrypha?; end
+
+  sig { returns(BibleBot::References) }
+  def chapters; end
+
+  sig { returns(T::Array[String]) }
+  def string_ids; end
+
+  sig { returns(T::Boolean) }
+  def single_full_chapter?; end
+
+  sig { returns(T::Boolean) }
+  def single_chapter?; end
+
+  sig { returns(T::Boolean) }
+  def single_booko?; end
```

Comment:
What's this?

Reply:
A typo. https://github.com/LittleLea/bible_bot/pull/19/commits/df3306c712e2f40400efc1bd9f415df2e1969995